### ISSUE TITLE
Touching sync status info in Hamburg menu closes the menu

### ISSUE THREAD

**Issue by alxndrsn** [medic/medic-webapp#3976]
This is frustrating for already-frustrated users who think that touching this information might trigger synching.

cc @samkanga @smbuthia

**Comment by ngaruko** [comment]
Looks like the menu still closes on clicking the `sync.now` link. Did you want to have a second look @wtekeu '?

[sync.mov.zip](https://github.com/medic/medic-webapp/files/2734789/sync.mov.zip)

**Comment by kennsippell** [comment]
This item was labeled fixed as part of #4805. I think the route of the issue herein is that clicking the item did nothing *and* closed the menu. The suggested fix from Alex is to _either_ make it do something (ideal), or have the menu stay open (at minimum). As described, I'd call this fixed.

> Ideally touching this information would provide some kind of indication that progress was being made, or trigger a new synch

@ngaruko - That said, I agree with you that it is an odd experience as-is because we use the content of a hidden menu to communicate the result of an action. The other actions in the menu close the menu and are not a problem because the result of those actions happen outside the menu. I personally think the best experience we could have here is that when somebody manually triggers a sync through the menu, we open a dialog which communicates details of the sync (current status, time of last sync, progress of current sync, a cancel button, detailed errors with troubleshooting tips, etc.). If folks agree, it seems best to open a new issue tracking that.

**Comment by garethbowen** [comment]
@kennsippell I agree that would be ideal. Can you please raise an issue for that improved feedback?

**Comment by kennsippell** [comment]
#5207
